<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>651</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="controlBlock">
        <item>
         <widget class="QWidget" name="volumeHolder" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QCheckBox" name="volumeEnable">
             <property name="text">
              <string>Volume</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="volumeSlider">
             <property name="maximum">
              <number>100</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="speedHolder" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>10</height>
           </size>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Speed</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="speedSlider">
             <property name="minimum">
              <number>-30</number>
             </property>
             <property name="maximum">
              <number>30</number>
             </property>
             <property name="singleStep">
              <number>0</number>
             </property>
             <property name="value">
              <number>0</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="tickPosition">
              <enum>QSlider::TicksBothSides</enum>
             </property>
             <property name="tickInterval">
              <number>10</number>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="buttonsHolder" native="true">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QToolButton" name="videoBack10Button">
             <property name="toolTip">
              <string>back 10 seconds</string>
             </property>
             <property name="text">
              <string>begin</string>
             </property>
             <property name="icon">
              <iconset>
               <normalon>:/icons/resources/begin.png</normalon>
              </iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="videoBackButton">
             <property name="toolTip">
              <string>back one frame</string>
             </property>
             <property name="text">
              <string>back1frame</string>
             </property>
             <property name="icon">
              <iconset>
               <normalon>:/icons/resources/back.png</normalon>
              </iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="videoPauseButton">
             <property name="toolTip">
              <string>pause</string>
             </property>
             <property name="text">
              <string>pause</string>
             </property>
             <property name="icon">
              <iconset>
               <normalon>:/icons/resources/pause.png</normalon>
              </iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="videoFwdButton">
             <property name="toolTip">
              <string>forward one frame</string>
             </property>
             <property name="text">
              <string>fwd1frame</string>
             </property>
             <property name="icon">
              <iconset>
               <normalon>:/icons/resources/fwd.png</normalon>
              </iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="videoPlayButton">
             <property name="toolTip">
              <string>play at normal speed</string>
             </property>
             <property name="text">
              <string>play</string>
             </property>
             <property name="icon">
              <iconset>
               <normalon>:/icons/resources/play.png</normalon>
              </iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="videoFwd10Button">
             <property name="toolTip">
              <string>forward 10 seconds</string>
             </property>
             <property name="text">
              <string>end</string>
             </property>
             <property name="icon">
              <iconset>
               <normalon>:/icons/resources/end.png</normalon>
              </iconset>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="videoBlock">
        <item>
         <widget class="QVideoWidget" name="videoWidget" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="videoPosition">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>651</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_New"/>
    <addaction name="action_Open"/>
    <addaction name="actionSave"/>
    <addaction name="action_Save_As"/>
    <addaction name="separator"/>
    <addaction name="actionSet_Media"/>
    <addaction name="action_Import_Funscript"/>
    <addaction name="action_Export_Funscript"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_New"/>
   <addaction name="action_Open"/>
   <addaction name="actionSave"/>
   <addaction name="action_Save_As"/>
   <addaction name="separator"/>
   <addaction name="actionSet_Media"/>
   <addaction name="action_Import_Funscript"/>
   <addaction name="action_Export_Funscript"/>
  </widget>
  <action name="action_New">
   <property name="icon">
    <iconset>
     <normalon>:/icons/resources/new.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;New</string>
   </property>
   <property name="toolTip">
    <string>create a new empty project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normalon>:/icons/resources/save.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>save current project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Open">
   <property name="icon">
    <iconset>
     <normalon>:/icons/resources/open.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="toolTip">
    <string>open existing project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save_As">
   <property name="text">
    <string>&amp;Save As...</string>
   </property>
   <property name="toolTip">
    <string>save as a new file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionSet_Media">
   <property name="icon">
    <iconset>
     <normalon>:/icons/resources/media.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>Set &amp;Media</string>
   </property>
   <property name="toolTip">
    <string>set the video file for the project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="action_Import_Funscript">
   <property name="icon">
    <iconset>
     <normalon>:/icons/resources/import.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Import Funscript</string>
   </property>
   <property name="toolTip">
    <string>imports a funscript into the project</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="action_Export_Funscript">
   <property name="icon">
    <iconset>
     <normalon>:/icons/resources/export.png</normalon>
    </iconset>
   </property>
   <property name="text">
    <string>&amp;Export Funscript</string>
   </property>
   <property name="toolTip">
    <string>exports the project to a funscript</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header location="global">QVideoWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
